AUDIO/PlayPain.md

```diff
@@ -5,27 +5,52 @@ ns: AUDIO
 
 ```c
 // 0xBC9AE166038A5CEC 0x874BD6CB
-void PLAY_PAIN(Ped ped, int painID, int p1);
+void PLAY_PAIN(Ped ped, int painID, float p3, bool createNetEvent);
```

Comment:
Adding/removing parameters is unfortunately not supported to keep C# backwards compatibility.

Reply:
Oh ok, left the same as they were